mgr.go

```diff
@@ -507,12 +523,15 @@ func (mgr *SysboxMgr) reqMounts(id, rootfs string, uid, gid uint32, shiftUids bo
 		containerMnts = append(containerMnts, m...)
 	}
 
-	// Add the linux kernel header mounts to the sys container. This is
-	// needed to build or run apps that interact with the Linux kernel
-	// directly within a sys container. Note that there is no need to
-	// track mntInfo for these since we are not backing these with
-	// sysbox-mgr data stores.
-	containerMnts = append(containerMnts, mgr.linuxHeaderMounts...)
+	// Add the linux kernel header mounts to the sys container. This is needed to
+	// build or run apps that interact with the Linux kernel directly within a
+	// sys container. Note that there is no need to track mntInfo for these since
+	// we are not backing these with sysbox-mgr data stores.
+	linuxHdrMount, err := getLinuxHeaderMounts(mgr.hostKernelHdrPath)
```

Comment:
We don't want to call getLinuxHeaderMounts() here, because it's a slow operation and will impact container startup time. We already called this function in newSysboxMgr(), so let's store the result in the sysboxMgr struct and use it here.

Reply:
This is definitely not intended, just a mistake on my part while merging the old & new code. Good catch!